src/main/java/edu/hm/hafner/analysis/parser/dry/cpd/CpdParser.java

```diff
@@ -64,6 +64,8 @@ protected Report convertDuplicationsToIssues(final List<Duplication> duplication
                         .setLineStart(file.getLine())
                         .setLineEnd(file.getLine() + duplication.getLines() - 1)
                         .setFileName(file.getPath())
+                        .setCategory("CPD")
```

Comment:
Move the category to https://github.com/jenkinsci/analysis-model/blob/a137c723b65b9eb93ceeb993ca35bf9b13363683/src/main/java/edu/hm/hafner/analysis/parser/dry/AbstractDryParser.java#L83

Reply:
I would prefer to set the category to „Code Duplication“.

Reply:
This is not yet addressed. Can't this be moved to the parent class?

Reply:
And „Code Duplication“ reads better than CamelCase.

Reply:
Should be good now. 
I replaced  `buildAndClean()` by `build()` in SimianParser and DupFinderParser otherwise from the second iteration in the [duplication.getBlocks() loop](https://github.com/emouty/analysis-model/blob/f90c316a6301722b304221c8123f532e0df73939/src/main/java/edu/hm/hafner/analysis/parser/dry/simian/SimianParser.java#L61) you would loose the consolidated fields ([message and category](https://github.com/emouty/analysis-model/blob/f90c316a6301722b304221c8123f532e0df73939/src/main/java/edu/hm/hafner/analysis/parser/dry/AbstractDryParser.java#L83))  from the AbstractDryParser. 
Didn't needed to do it in CPDParser as it already used `build()`.